{{crane/gcrane}}: add documentation pages

rvandernoort opened the pull request:
<!--
Thank you for contributing!
Please fill in the following checklist, removing items that do not apply.
See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
-->

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**
  - crane: v0.20.2
  - gcrane: v0.20.1

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/tldr-pages/tldr?pullRequest=13853) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/tldr-pages/tldr?pullRequest=13853) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/tldr-pages/tldr?pullRequest=13853) it.</sub>

rvandernoort commented on pages/common/gcrane-completion.md:
```diff
@@ -0,0 +1,29 @@
+
+`source <(gcrane completion bash)>`
+
+- Load completions for every new sesion:
```
```suggestion
- Load completions for every new session:
```

rvandernoort commented:
Hi thanks for the extensive review. I'll implement your feedback after the weekend and then we can hopefully merge soon after! Cheers

rvandernoort commented:
Hi I've updated the PR according to the guidelines and your feedback can you take another look? Furthermore, as the bot indicates I've added the various implementations of `gcrane completion` which has some OS specific commands, I hope I correctly implemented that aspect.

rvandernoort commented on pages/common/gcrane.md:
```diff
@@ -0,0 +1,32 @@
+
+- Execute a `gcrane` subcommand:
+
+`gcrane [flags] {{subcommand}}`
```
All the global flags are all mentioned in this file, but I know how to make it more clear.

rvandernoort replied:
So I changed both `crane` and `gcrane` pages:
- first entry is without flags just referring to the subcommands in the description
- other entries contain the global flags and now also include the `{{subcommand}}` entry as they can be used anywhere.

rvandernoort commented:
Ah yes sorry, I though rebasing the branch with main was the way to go, but I get that this complicates the review. I've updated your last comments.

nelsonfigueroa reviewed (approved):
Looks good to me. Thanks for the huge contribution!
